formhero/forms/tests.py

```diff
@@ -1,4 +1,24 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from formhero.apps.models import App
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from formhero.forms.views import CreateForm
+from rest_framework import status
 
 
-# Create your tests here.
+User = get_user_model()
+
+
+class RetrievePostFormTests(APITestCase):
```

Comment:
@szymonzmyslony I think we want to name this `CreateFormTests`, there aren't any tests for retrieving yet. In fact...

1. The view does not yet support retrieving objects.
2. It never will: in DRF, one view tends to handle the "list" and "create" operations for a resource/model. A separate view handles "retrieve", "update" and "destroy" for individual objects.
3. We would tend to write separate test classes for "create" and "list" anyway.

Familiarise yourself with the verbs DRF uses for it's mixins and generic views (list create retrieve update destroy), and stick to them when naming stuff for yourself. It will make the codebase more consistent and easy to read over time. http://www.django-rest-framework.org/api-guide/generic-views/#mixins